.github/workflows/test.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       - uses: taiki-e/install-action@cargo-hack
       - uses: Swatinem/rust-cache@v2
       - run: |
-          cargo hack check --feature-powerset --optional-deps serde,rkyv \
+          cargo hack check --feature-powerset --optional-deps serde,rkyv-validation \
```

Comment:
```suggestion
          cargo hack check --feature-powerset --optional-deps serde \
```

This fails on CI.

Reply:
I reverted this to `,rykv` instead; seeing if that passes.

Reply:
As I understand it we only need to put things `--optional-deps` that are not named as features. I tested it locally and `cargo hack` does the combinations we want without `,rkyv`.

Reply:
And with `,rkyv` it tests double the number of combinations...